package-res/resources/web/prompting/api/EventAPI.js

```diff
@@ -79,13 +79,25 @@ define([], function() {
      * @method
      * @param {Function} callback   The function to be executed when the event is triggered.
      *                              Pass null if you wish to unbind this event
+     * @param {String} paramName    The name of the parameter to which the callback will be called
+     *                              This argument is optional. If no parameter name is passed on
+     *                              the callback will be assigned to all the parameters.
      * @example
      *  api.event.parameterChanged(function(parameterName, parameterValue) {
      *    // Execute event based code
      *  })
      */
```

Comment:
Also we can add one more example (doc ) for registering callback for concrete parameter name.